base/sparse/cholmod.jl

```diff
@@ -122,6 +133,15 @@ function __init__()
              the correct versions of all dependencies.
          """)
     end
+
+    ### Initiate CHOLMOD
+    global const cmn = fill(0xff, common_size)
+    start(cmn)
+
+    cnfg = cglobal((:SuiteSparse_config, :libcholmod), Ptr{Void})
```

Comment:
This symbol is in `libsuitesparseconfig`, at least with recent versions of suitesparse. I don't know where it is, if anywhere, on older versions. This may mean we need to bump the minimum suitesparse version requirement, cc @nalimilan.